src/Agent.Worker/Release/ContainerFetchEngine/BackoffTimerHelper.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release.ContainerFetchEngine
+{
+    public static class BackoffTimerHelper
+    {
+        public static TimeSpan GetExponentialBackoff(int attempt, TimeSpan minBackoff, TimeSpan maxBackoff, TimeSpan deltaBackoff)
```

Comment:
Can you add a short comment about the backoff algorithm? Btw, do we really need this kind of complex way to figure out backoff timing?